build_tools/cmake/iree_cc_test.cmake

```diff
@@ -160,6 +160,25 @@ function(iree_cc_test)
         TEST_TMPDIR=${_ANDROID_ABS_DIR}/test_tmpdir
     )
     set_property(TEST ${_NAME_PATH} PROPERTY ENVIRONMENT ${_ENVIRONMENT_VARS})
+  elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv" AND RISCV_CPU STREQUAL "rv64")
+    # The test target needs to run within the QEMU emulator for RV64 Linux
+    # crosscompile build. A QEMU 64 Linux emulator must be available at the
+    # path specified by the `QEMU_RV64_BIN` environment variable.
+    if(DEFINED ENV{QEMU_RV64_BIN})
+      add_test(
+        NAME
+          ${_NAME_PATH}
+        COMMAND
+          "$ENV{QEMU_RV64_BIN}"
+          -cpu rv64,x-v=true,x-k=true,vlen=512,elen=64,vext_spec=v1.0
```

Comment:
Yeah I think this could cause a problem with building this on CI. We might be building this on a machine/environment that doesn't have qemu (or doesn't have the env var set) and then moving it to one that does. Silently not creating the test seems like a bad option. If nothing else, delegating to a script matches what we do for Android, so it would at least be consistent between them.

Reply:
Move QEMU command into `run_riscv64_test.sh`. Add a TODO for @dcaballe to add the on-device run command.